processor/compile/compatability.ts

```diff
@@ -0,0 +1,25 @@
+import { Html } from 'mdast';
+import { NodeTypes } from '../../enums';
+
+type CompatNodes =
+  | { type: NodeTypes.variable; text: string }
+  | { type: NodeTypes.glossary; data: { hProperties: { term: string } } }
+  | { type: NodeTypes.reusableContent; tag: string }
+  | Html;
+
+const compatability = (node: CompatNodes) => {
+  switch (node.type) {
+    case NodeTypes.variable:
+      return `<Variable name="${node.text}" />`;
+    case NodeTypes.glossary:
+      return `<Glossary>${node.data.hProperties.term}</Glossary>`;
+    case NodeTypes.reusableContent:
+      return `<${node.tag} />`;
+    case 'html':
+      return node.value.replaceAll(/<!--(.*)-->/g, '{/*$1*/}');
+    default:
+      throw new Error('Unhandled node type!');
+  }
+};
+
+export default compatability;
```

Comment:
😬 😬 😬 how do i break it to you that you spelled compatibility wrong 😬

Reply:
If spelling compatability like compat-ability is wrong, I don't want to be right.